src/ScottPlot4/ScottPlot.Demo/ScottPlot.Demo.WinForms/WinForms Demo.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <OutputType>WinExe</OutputType>
-        <TargetFramework>net48</TargetFramework>
+		<TargetFrameworks>net6.0-windows</TargetFrameworks>
```

Comment:
I don't think we should change the target platform here: The demo applications intentionally target .NET Framework because I want them to be small and simple to run and more Windows computers have it already.